spec/dato/client_spec.rb

```diff
@@ -66,4 +66,11 @@
       expect(response.code).to eq(404)
     end
   end
+
+  describe "upload" do
+    it "can upload an image" do
+      job_id = client.uploads.create_from_url(url: "https://upload.wikimedia.org/wikipedia/commons/4/4f/SVG_Logo.svg")
+      expect(job_id).to be_a(String)
```

Comment:
can you extend the test to expect `{ upload_id: '67094498' }`. since we use VCR it can be a fixed number